substreams-helper/build.rs

```diff
@@ -2,6 +2,8 @@ use anyhow::{Ok, Result};
 use substreams_common::codegen;
 
 fn main() -> Result<(), anyhow::Error> {
+    codegen::generate(None)?;
     codegen::generate_abi(None)?;
```

Comment:
Remove this

Reply:
How are the erc20 abis retrieved then? The erc20 abi is needed for the erc20 functions in `substreams-helper`